packages/uswds-core/src/js/utils/validate-input.js

```diff
@@ -14,20 +14,52 @@ module.exports = function validate(el) {
     throw new Error(`No validation element found with id: "${id}"`);
   }
 
+  let statusSummary = "";
   Object.entries(el.dataset).forEach(([key, value]) => {
     if (key.startsWith("validate")) {
       const validatorName = key.substr("validate".length).toLowerCase();
       const validatorPattern = new RegExp(value);
       const validatorSelector = `[data-validator="${validatorName}"]`;
       const validatorCheckbox = checkList.querySelector(validatorSelector);
+      const validatorParent = el.parentNode;
+      const statusSummaryContainer = validatorParent.querySelector(`[data-validation-status]`);
+
+      const checked = validatorPattern.test(el.value);
+      validatorCheckbox.classList.toggle(CHECKED_CLASS, checked);
 
       if (!validatorCheckbox) {
         throw new Error(`No validator checkbox found for: "${validatorName}"`);
       }
 
-      const checked = validatorPattern.test(el.value);
-      validatorCheckbox.classList.toggle(CHECKED_CLASS, checked);
-      validatorCheckbox.setAttribute(CHECKED, checked);
+      // Create status reports for checklist items
+      let statusComplete = "status complete";
+      let statusIncomplete = "status incomplete";
+      let checkboxContent = `${validatorCheckbox.textContent} `;
+
+      if (el.hasAttribute("data-validation-complete")) {
+        statusComplete = el.getAttribute("data-validation-complete");
+      }
+
+      if (el.hasAttribute("data-validation-incomplete")) {
+        statusIncomplete = el.getAttribute("data-validation-incomplete");
+      }
+
```

Comment:
We could simplify this by having these labels as a backup in case the dataset comes back null or undefined.

```suggestion
      const statusComplete = el.dataset.validationComplete || "status complete";
      const statusIncomplete = el.dataset.validationIncomplete || "status incomplete";
```

Reply:
Clean! Updated in [aea87ce](https://github.com/uswds/uswds/pull/4914/commits/aea87ce06c60429928e9ff842b06531e7e9ca202)